src/test/resources/org/broadinstitute/hellbender/tools/ClipReads/expected.clippingReadsTest.withRG.hg19.QT_10_CT_15_X_CCCCC_XF.tmp

```diff
@@ -8,7 +8,7 @@ Percent of clipped bases              24.70
 Number of quality-score clipped bases 126
 Number of range clipped bases         65
 Number of sequence clipped bases      53
-         8 clip sites matching GAGG
         10 clip sites matching CCCCC
+         8 clip sites matching GAGG
```

Comment:
yes, the only change was to resort those guys (which makes more sense then list them in an arbitrary order)